src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,135 @@
+import React, { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types/Person';
+import { getPeople } from '../api';
+import { SortField } from '../types/SortField';
 
 export const PeoplePage = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState('');
+  const [searchParams, setSearchParams] = useSearchParams();
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('century');
+
+  const loadData = async () => {
+    try {
+      const people = await getPeople();
+
+      if (people.length === 0) {
```

Comment:
```suggestion
      if (!people.length) {
```

can be a bit simplify